clients/python/test/e2e/data/server_client_compatibility.json

```diff
@@ -0,0 +1 @@
+{"master":{"api.osparc-master.speag.com":true,"api.osparc-staging.speag.com":false,"api.osparc.speag.com":false,"api.staging.osparc.io":false,"api.osparc.io":false},"production":{"api.osparc-master.speag.com":true,"api.osparc-staging.speag.com":true,"api.osparc.speag.com":true,"api.staging.osparc.io":true,"api.osparc.io":true}}
```

Comment:
- MINOR: format
- TIP: I would make the json  config more self-explanatory. It adds affordable complexity and it is easily extendable.  e.g.
```json
[
  {
    "name": "master": {
      "url": "api.osparc-master.speag.com",
      "is_compatible": true,
      "url": "api.osparc-staging.speag.com",
      "is_compatible": false,
      "url": "api.osparc.speag.com",
      "is_compatible": false,
      "url": "api.staging.osparc.io",
      "is_compatible": false,
      "url": "api.osparc.io",
      "is_compatible": false
    }
  },
  {
    "name": "production": {
      "url": "api.osparc-master.speag.com",
      "is_compatible": true,
      "url": "api.osparc-staging.speag.com",
      "is_compatible": true,
      "url": "api.osparc.speag.com",
      "is_compatible": true,
      "url": "api.staging.osparc.io",
      "is_compatible": true,
      "url": "api.osparc.io",
      "is_compatible": true
    }
  }
]
```

Reply:
I agree to this change. However, I have a feeling I will work more on this in the future, so in order to advance on the tutorial I would prefer to leave this as it is and do it some other time. OK for you?

Reply:
done